app/app_management_strings.grdp

```diff
@@ -85,4 +85,10 @@
   <message name="IDS_APP_MANAGEMENT_STORAGE" desc="Label for the Android storage permission toggle.">
     Storage
   </message>
+  <message name="IDS_APP_MANAGEMENT_SYSTEM_APP_POLICY_STRING" desc="Tooltip label explaining that an app cannot be uninstalled as it is a part of Brave OS (the operating system).">
+    This app cannot be uninstalled as it is part of Brave OS.
```

Comment:
I don't think this string is used anywhere; but looks like the Chrome/Brave/g replacement got this one (ex: we might exclude `Chrome` if there's a lookahead for ` OS`). But since it's probably not used, it's fine as-is IMO

Reply:
ah 👍